ota-plus-web/app/reactapp/src/components/updates/createWizard/Step1.jsx

```diff
@@ -92,6 +95,7 @@ class Step1 extends Component {
                   );
                 })
               )}
+              <div className="information-text">{t('updates.creating.wizard.information_step1')}</div>
```

Comment:
This should be displayed on when user has one ECU. Additionally it should be not a part of scroll list.

Reply:
done